news/sub-proc-tracer.rst

```diff
@@ -0,0 +1,24 @@
+**Added:**
+
+* new environment variable ``$XONSH_SUBPROC_TRACER``
```

Comment:
I'd suggest that the variable name be something like `XONSH_TRACE_SUBPROC_RUNNER` -- it's a bit awkward and I'm open to suggestions, but since it modifies / extends the behavior of a different env-var, it's better if they're clustered alphabetically

Reply:
I have used `XONSH_TRACE_SUBPROC_FUNC` since `RUNNER` sounds like handling the subprocess call.